### PR TITLE
System time boxes

### PR DESCRIPTION
**Pull request by kmcostel**
System time boxes appear now below the text area when the edit button is clicked. Not entirely useful/functional, but when a document is clicked the text of all documents turns blue. Added a selected document label above the edit/view/delete buttons.

### CODE REVIEW

File: public/bitemp_graph.js [hunk 1]
```diff
@@ -164,20 +184,31 @@ var barChart = function() {
         .enter()
         .append('g')
         .attr('class','split');
-
+      
+      var strokeColor = 'black';
+      
+      /*for (var prop in datum) {
```
Comment (dmcassel) [inline public/bitemp_graph.js]: dead code; please remove

File: public/bitemp_graph_helper.js [hunk 2]
```diff
@@ -8,17 +8,24 @@ function fillText(data, isEditing) {
...
         data[property] = null;
       }
-      if (textArea.value === '{') {
-        textArea.value += '\n\"' + property + '\": ' + '\"'+ data[property] +'\"';
+      if (textArea.value !== '{') { //Add a comma onto previous line, if not on the first item.
+        strToAdd += ',';
       }
-      else {
-        textArea.value += ',\n\"' + property + '\": ' + '\"'+ data[property] +'\"';
+      strToAdd += '\n\"' + property + '\": '; //+ '\"'+ data[property] +'\"';
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: looks like the comment at the end should be removed

File: public/bitemp_graph_helper.js [hunk 3]
```diff
@@ -147,12 +161,34 @@ var removeButtonEvents = function () {
...
   $('#saveButton').unbind('click');
   $('#change-prop').unbind('click');
+};
+
+var drawChart = function (params, docProp) {
+  var chart = barChart()
+    .data(params.data)
+    .width(params.width)
+    .height(params.height)
+   // .valStart(params.valStart)
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: dead code?

File: public/search.js [hunk 4]
```diff
@@ -46,9 +46,39 @@ $('#search').click(function()
     $('#next').css({'visibility': 'visible'});
     $('#prev').css({'visibility': 'visible'});
     displayDocs(firstDoc, lastDoc);
+    
+    var dropDownList = document.getElementById('dropdown');
+    var selectedColl = dropDownList.options[dropDownList.selectedIndex].value;
+    $.ajax(
+    {
+      //url: 'http://localhost:3000/v1/resources/temporal-range?rs:collection='+selectedColl,
+      url: 'http://localhost:3000/v1/resources/temporal-range?rs:collection=myTemporal' ,
```
Comment (dmcassel) [inline public/search.js]: Why the switch to hard-coded collection name instead of using the selectedColl?

File: public/search.js [hunk 5]
```diff
@@ -46,9 +46,39 @@ $('#search').click(function()
...
 );
 
+//function 
+function displayAxis(times)
+{
+  sysStart = times.sysStart;
+  sysEnd = times.sysEnd;
+  valStart = times.valStart;
+  valEnd = times.valEnd;
+  console.log(sysStart);
```
Comment (dmcassel) [inline public/search.js]: If these console.log statements aren't needed any more, please remove them. If they are, please add something to the output to give some context. console.log('displayAxis() sysStart: ' + sysStart);

File: public/bitemp_graph_helper.js (hunk 3 above)
Comment (kmcostel) [inline public/bitemp_graph_helper.js]: It's eventually going to be added with Ashley and Hilary's code, they just don't have it functioning yet.

File: public/search.js (hunk 4 above)
Comment (kmcostel) [inline public/search.js]: Was trying to debug, will change back

**Pull request merged**